deploy/charts/secret-manager/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: secret-manager
 description: External secret management for Kubernetes
 type: application
 version: 0.1.2
-appVersion: 0.0.0
+appVersion: "%%APP_VERSION%%"
```

Comment:
The chart will not lint like this. It needs to be semver compatible which is why it was 0.0.0

Reply:
The workflow fixes this before lint and test, which is why it passed on this branch. Also locally, `helm lint` doesn't seem to mind this, only `version` is checked against semver, `appVersion` is just checked if it is a string ([source](https://github.com/helm/helm/blob/e1d4c674375e724d6cf72c91219986cc453f4c76/pkg/lint/rules/chartfile.go#L59))